Fix the build

```
pipeline.exceptions.CompressorError: b'\n/usr/lib/node_modules/yuglify/bin/yuglify:77\n                throw(err);\n                ^\nSyntaxError: Unexpected token: name (postal_address_does_differ)\n    at JS_Parse_Error.get (eval at <anonymous> (/usr/lib/node_modules/yuglify/node_modules/uglify-js/tools/node.js:20:1), <anonymous>:71:23)\n'
Makefile:114: recipe for target 'collectstatic' failed
make: *** [collectstatic] Error 1
The command "make build up wait-for-db generate-django-secret migrate collectstatic selenium-up" exited with 2.
```

See #300

--- a/.travis.yml
+++ b/.travis.yml
@@ -36,7 +36,8 @@ jobs:
       script:
         - pushd deployment
         - echo "Preparing Docker Stack"
-        - make build up wait-for-db generate-django-secret migrate collectstatic selenium-up
+        - make build up wait-for-db generate-django-secret migrate collectstatic
+        - make selenium-up
         - make status
         - echo "Docker stack ready"
         - echo "Preparing unittests"

